src/mbgl/util/mapbox.cpp

```diff
@@ -85,6 +95,11 @@ std::string normalizeGlyphsURL(const std::string& url, const std::string& access
     }
 
     std::vector<std::string> pathname = getMapboxURLPathname(url);
+
+    if (pathname.size() < 3) {
```

Comment:
In this particular case, `isDraft` assumes a pathname longer than 3. That’s dead code now.